src/queries/hooks/rewards/useSendRecentActivities.ts

```diff
@@ -0,0 +1,9 @@
+import { useMutation } from '@tanstack/react-query';
+import { sendUserRecentActivities } from 'queries/queryKeys';
+import { sendRecentActivities } from 'queries/services';
+
+export const useSendRecentActivities = (payload: { userId: string }) =>
```

Comment:
Don't think this is a correct name for the hook. Please fix the name.

Reply:
Updated naming to getRewardActivityStatus - since it just returns a list of items that i can use to check daily rewards status and locked status